starter-code/scripts/article.js

```diff
@@ -22,22 +31,24 @@ Article.prototype.toHtml = function() {
   $newArticle.find('time[pubdate]').attr('title', this.publishedOn);
   $newArticle.find('time').html('about ' + parseInt((new Date() - new Date(this.publishedOn))/60/60/24/1000) + ' days ago');
 
-  /* TODO: This clone article is no longer a template,
+  /* This clone article is no longer a template,
   as it now has real data attached to it! We need to account
   for that before this current article gets rendered to our
   DOM. */
 
+  $('#article').append($newArticle);
```

Comment:
You can omit this as the last operation below traverses through the array of objects and appends each one to the #articles.